app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -152,13 +152,14 @@ KT.hosts.onKatelloHostEditLoad = function(){
 
     $.each(prefixes, function(index, prefix) {
         $.each(attributes, function(attrIndex, attribute) {
-            $('body').on('select2:select select2:unselecting', '#' + prefix + '_' + attribute, function () {
+            $('body').on('select2:select.hostsContentSourceNS select2:unselecting.hostsContentSourceNS', '#' + prefix + '_' + attribute, function () {
                 KT.hosts.toggle_installation_medium();
             });
         });
     });
 
-    $('body').on('select2:select select2:unselecting', '#content_source_id', function () {
+    $('body').off('.hostsContentSourceNS');
```

Comment:
Now that you've addressed the other events on 155, should this `.off` move up to 152 before that .each ?